app/templates/frameworks/_framework_actions.html

```diff
@@ -27,17 +27,11 @@
 <li class="browse-list-item">
   <a class="browse-list-item-link"
      href="{{ url_for('.supplier_details') }}">
-    <span>Enter your company details</span>
+    <h2>1. Enter your company details</h2>
```

Comment:
The indentations here seems a bit off. For example lines 27, 37, 38...

Reply:
Our indentations are generally screwy within the templates. Turn off whitespace diffs might help.

Reply:
It's the same in the browser. It's not that straightforward to fix up though...

Reply:
Oh hmm. I'll take another look at it in the browser. I thought it looked fine when I did it 🤔  Maybe I didn't look closely enough.

Reply:
Do you mean visually something on the page looks off, or in the source code the elements are indented a different amount? If it's the latter, that's not something we really seem to care about (certainly from what I've seen all over the rest of the code base)

Reply:
Sorry, wrong word. I mean in the editor (as in turning off whitespace won't help). Don't worry about it, it's not a big deal.